### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by kuroko137**

### CODE REVIEW

File: resources/l10n/ja/LC_MESSAGES/loot.po [hunk 1]
```diff
@@ -7,289 +7,458 @@ msgid ""
 msgstr ""
 "Project-Id-Version: LOOT 0.10.3\n"
```
Comment (pStyl3) [inline resources/l10n/ja/LC_MESSAGES/loot.po:8]: Please update the Project-Id-Version according to point 5 [here](https://github.com/loot/loot/blob/master/CONTRIBUTING.md#translating-the-loot-application) to `LOOT 0.18.2`.

File: resources/l10n/ja/LC_MESSAGES/loot.po [hunk 2]
```diff
@@ -301,657 +470,796 @@ msgstr ""
...
+#~ msgstr ""
+#~ "画面上の項目の多くにはヒントが用意されています。どういう機能か知りたくなっ"
+#~ "たらカーソルを載せて調べてみてください。また、メニューからLOOTのマニュアル"
+#~ "を開くこともできます。"
+
+#~ msgid "OK"
+#~ msgstr "OK"
 
 #~ msgid "Error: Could not initialise logging."
 #~ msgstr "エラー: ログの出力を開始できません。"
```
Comment (pStyl3) [inline resources/l10n/ja/LC_MESSAGES/loot.po:1265]: Please delete these old & outcommented translations, using `Translation > Purge deleted translations > Purge` in Poedit.

**Review by pStyl3 — CHANGES_REQUESTED** [review]

File: resources/l10n/ja/LC_MESSAGES/loot.po (hunk 2 above)
Comment (kuroko137) [reply resources/l10n/ja/LC_MESSAGES/loot.po]: Thanks for the review. I have fixed those.

**Review by pStyl3 — APPROVED** [review]

**Comment by Ortham** [comment]
Thanks! 👍

**Pull request merged**